packages/core/src/lib/wallet/wallet.types.ts

```diff
@@ -121,6 +128,112 @@ interface SignAndSendTransactionsParams {
   transactions: Array<Optional<Transaction, "signerId">>;
 }
 
+interface SignTransactionParams {
+  /**
+   * The Transaction object to sign
+   */
+  transaction: Tx;
+  /**
+   * The {Signer} object that assists with signing keys
+   */
+  signer: Signer;
```

Comment:
How do you envision the use of it? Isn't the whole point to use an external wallet as signer? How is it required?